src/plugins/timeBarAllActivities/index.tsx

```diff
@@ -40,7 +40,7 @@ export default definePlugin({
     settings,
     patches: [
         {
-            find: ".Messages.USER_ACTIVITY_PLAYING",
+            find: 'location:"UserProfileActivityCard"',
```

Comment:
```suggestion
            find: ".gameState,children",
```
Location stuff is related to analytics and more likely to be removed in the future

Reply:
Didn't know that, will keep that in mind